src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -0,0 +1,108 @@
+    [contact-list-item/contact-list-item
+     {:on-press      #(rf/dispatch [:chat.ui/show-profile public-key])
+      :on-long-press show-profile-actions
+      :accessory     {:type     :options
+                      :on-press show-profile-actions}}
+     {:primary-name   primary-name
+      :secondary-name secondary-name
+      :public-key     public-key
+      :ens-verified   ens-verified
+      :added?         added?}
+     {:public-key public-key}]))
+
+(defn- footer
+  []
+  [rn/view {:style style/footer}])
+
+(defn- get-item-layout
+  [_ index]
+  #js {:length 200 :offset (* 200 index) :index index})
+
+(defn- members
+  [items]
+  [rn/section-list
+   {:key-fn                            :public-key
+    :content-container-style           {:padding-bottom 20}
+    :get-item-layout                   get-item-layout
+    :content-inset-adjustment-behavior :never
+    :sections                          items
+    :sticky-section-headers-enabled    false
+    :render-section-header-fn          contact-list/contacts-section-header
+    :render-section-footer-fn          footer
+    :render-fn                         contact-item
+    :scroll-event-throttle             8}])
+
+(defn- view-internal
+  [_args]
+  (fn [{:keys [theme]}]
+    (let [{:keys [chat-id community-id]} (rf/sub [:get-screen-params :view-channel-members-and-details])
+          {:keys [description chat-name emoji muted chat-type]
+           :as   chat}                   (rf/sub [:chats/chat-by-id chat-id])
+          pins-count                     (rf/sub [:chats/pin-messages-count chat-id])
+          items                          (rf/sub [:communities/sorted-community-members-section-list
+                                                  community-id])
+          profile-color                  (rf/sub [:profile/customization-color])]
+      [rn/view {:style {:flex 1}}
+       [quo/page-nav
+        {:background :blur
+         :icon-name  :i/arrow-left
+         :on-press   #(rf/dispatch [:navigate-back])
+         :right-side [{:icon-name :i/options
+                       :on-press  #(rf/dispatch [:show-bottom-sheet
+                                                 {:content (fn [] [home.actions/chat-actions
+                                                                   chat
+                                                                   false
+                                                                   true])}])}]}]
+       [quo/text-combinations
+        {:container-style                 style/text-combinations
+         :title                           [quo/channel-name
+                                           {:channel-name chat-name
+                                            :unlocked?    true}]
+         :emoji                           (when (not (string/blank? emoji)) emoji)
+         :emoji-background-color          (colors/resolve-color profile-color theme 10)
```

Comment:
please do this in the component and just pass `customization-color`. We want the design system to encapsulate styles so we can configure it by props :)

Reply:
Also it's nice if our quo components have a consistent api so we know what to expect.